internal/logger/logger.go

```diff
@@ -39,29 +45,33 @@ func (l *Logger) Printf(format string, a ...any) {
 	l.printf(format, a...)
 }
 
-func (l *Logger) UpgradeStarting(guid string) {
+func (l *Logger) UpgradeStarting(name string, guid string) {
```

Comment:
pickiest of nitpicks you could just declare the type once ie. (name, guid string)

Reply:
tell that to Goland refactoring tool 😝
Thanks for reminding me - it was on my list of things to change but then I forgot.